lib/src/models/comment_forest.dart

```diff
@@ -98,11 +98,20 @@ class CommentForest {
   /// (default: 32), and [threshold] is the minimum number of comments that a
   /// [MoreComments] object needs to represent in order to be expanded (default:
   /// 0).
-  Future<void> replaceMore({limit = 32, threshold = 0}) async {
+  ///
+  /// If [specificMoreComments] is provided replaceMore() will only expand the given
+  /// [MoreComments] and return after that one has been expanded
+  Future<void> replaceMore(
+      {limit = 32, threshold = 0, MoreComments specificMoreComments}) async {
```

Comment:
`specificMoreComments` is probably a bit too verbose. How about we go with `specific` instead?